src/include/elektra.h

```diff
@@ -0,0 +1,56 @@
+/**
+ * @file
+ *
+ * @brief Elektra High Level API.
+ *
+ * @copyright BSD License (see doc/LICENSE.md or http://www.libelektra.org)
+ */
+
+#ifndef ELEKTRA_H
+#define ELEKTRA_H
+
+#include "kdb.h"
+#include "kdbhelper.h"
+#include "kdbtypes.h"
+
+struct _ElektraError
+{
+    int errorNr;
```

Comment:
Why is this needed? Do you want to change elektraHasError to return a kdb_long_t?